taprpc/taprootassets.proto

```diff
@@ -348,6 +348,25 @@ message GenesisReveal {
     GenesisInfo genesis_base_reveal = 1;
 }
 
+message DecimalDisplayOpt {
```

Comment:
nit: why "Opt"? And not just DecimalDisplay?

Reply:
because it's optional, as in "it might be there"

Reply:
Yeah, but that is already signified by a child member of that message. So I think it's not necessary?

Reply:
ok, changing